plugins/woocommerce/src/Internal/Logging/RemoteLogger.php

```diff
@@ -470,6 +485,26 @@ function ( $trace_item ) {
 		return implode( "\n", $sanitized_trace );
 	}
 
+
+	/**
+	 * Redact potential user data from the message.
+	 *
+	 * @param string $message The message to redact.
+	 * @return string The redacted message.
+	 */
+	private function redact_user_data( $message ) {
```

Comment:
I'm wondering if we should make this a filter so that we can add more arbitrary sanitisation? We'd have to make sure that the wp filter code has been loaded by this point though

Reply:
If we're sanitising PII we should probably include credit card numbers too, since those are quite likely to pop up in woocommerce context

Reply:
That's a good idea! I'll add that.

Reply:
Updated the method to sanitise credit card numbers as well. 0100bf4a4f8bd890a2929386e7acdbe8d2a03c6e